src/helpers.js

```diff
@@ -32,40 +32,42 @@ const CONSOLE_API_KEYS = {
  * @returns {any} Returns a config object or null
  */
 async function getCommerceAdminConfig() {
-	const configFile = Config.get('aio-cli-plugin-commerce-admin');
+	const configFile = Config.get('mesh-api-manager.configPath');
 
 	if (!configFile) {
 		return {
-			baseUrl: 'https://graph.adobe.io/api-admin',
+			baseUrl: 'https://developers.adobe.io/console',
```

Comment:
Checked with Roman, and this will be the PROD URL where the dev console API will be hosted when deployed.

Reply:
Ah. ok. I was getting 404 when tried to access "https://developers.adobe.com/console" . but our existing prod dev console url doesn't have that "s" at the end and that is working. Wanted to check.